app/javascript/dashboard/modules/widget-preview/components/Widget.vue

```diff
@@ -32,52 +76,182 @@ export default {
       default: '',
       required: true,
     },
-    websiteDomain: {
-      type: String,
-      default: '',
-    },
     logo: {
       type: String,
       default: '',
     },
-    isExpanded: {
-      type: Boolean,
-      default: true,
-    },
     isOnline: {
       type: Boolean,
       default: true,
     },
     replyTime: {
       type: String,
-      default: 'few minutes',
+      default: 'in_a_few_minutes',
+    },
+    color: {
+      type: String,
+      default: '#1f93ff',
+    },
+    widgetBubblePosition: {
+      type: String,
+      default: 'left',
+    },
+    widgetBubbleLauncherTitle: {
+      type: String,
+      default: 'Chat with us',
+    },
+    widgetBubbleType: {
+      type: String,
+      default: 'standard',
     },
   },
+  data() {
+    return {
+      widgetScreens: [
+        {
+          id: 'default',
+          title: this.$t('INBOX_MGMT.WIDGET_BUILDER.WIDGET_SCREEN.DEFAULT'),
+          checked: true,
+        },
+        {
+          id: 'chat',
+          title: this.$t('INBOX_MGMT.WIDGET_BUILDER.WIDGET_SCREEN.CHAT'),
+          checked: false,
+        },
+      ],
+      isDefaultScreen: true,
+      isWidgetVisible: true,
+    };
+  },
   computed: {
     getWidgetHeadConfig() {
       return {
         welcomeHeading: this.welcomeHeading,
-        welcomeTagLine: this.welcomeTagLine,
+        welcomeTagline: this.welcomeTagline,
         websiteName: this.websiteName,
-        websiteDomain: this.websiteDomain,
         logo: this.logo,
-        isExpanded: this.isExpanded,
+        isDefaultScreen: this.isDefaultScreen,
         isOnline: this.isOnline,
-        replyTime: this.replyTime,
+        replyTime: this.replyTimeText,
+        color: this.color,
       };
     },
+    getWidgetBodyConfig() {
+      return {
+        isDefaultScreen: this.isDefaultScreen,
+        isOnline: this.isOnline,
+        replyTime: this.replyTimeText,
+        color: this.color,
+      };
+    },
+    getWidgetFooterConfig() {
+      return {
+        isDefaultScreen: this.isDefaultScreen,
+        color: this.color,
+      };
+    },
+    replyTimeText() {
+      switch (this.replyTime) {
+        case 'in_a_few_minutes':
+          return this.$t(
+            'INBOX_MGMT.WIDGET_BUILDER.REPLY_TIME.IN_A_FEW_MINUTES'
+          );
+        case 'in_a_few_hours':
```

Comment:
I think you can remove this from the `case` as it seems to be the default case.

Reply:
Done